scripts/templates/field_details.j2

```diff
@@ -25,7 +25,12 @@ Find additional usage and examples in the {{ fieldset['name'] }} fields <<ecs-{{
 {% if 'original_fieldset' not in field -%}
 
 {# `Field` column -#}
+{#- Beta fields will add the `beta` label -#}
+{%- if field['beta'] -%}
```

Comment:
I'm surprised this doesn't fail with a KeyError. Is Jinja2 more lenient dictionary access?